eclass/coreos-kernel.eclass

```diff
@@ -132,8 +132,10 @@ shred_keys() {
 }
 
 coreos-kernel_src_unpack() {
+	# we more or less reproduce the layout in /lib/modules/$(uname -r)/
 	mkdir -p "${S}/build" || die
-	ln -s "${KERNEL_DIR}"/* "${S}/" || die
+	mkdir -p "${S}/source" || die
```

Comment:
Since you are switching from a nested for to side-by-side can source be a symlink instead of a dir of symlinks now?